tests/robustness/coverage/coverage_test.go

```diff
@@ -30,6 +33,88 @@ import (
 	"google.golang.org/protobuf/encoding/protojson"
 )
 
+type column struct {
+	name string
+
+	matcher Match
+	modeler Model
+}
+
+type methodMather struct {
+	name string
+
+	matcher Match
+}
+
+type refOp struct {
+	// args splits traces into buckets based on Matcher or Modeler results.
+	args []column
+	// methods tries to associate Matched with method name.
+	methods []methodMather
```

Comment:
`methodMather`?

Reply:
Renamed to just `method` so it will appear as `methods []method` here